test/conftest.py

```diff
@@ -31,6 +31,7 @@
 import time
 import traceback
 
+import _pytest
```

Comment:
I'm guessing this is a private module for a reason... what's the situation with `_pytest`? Should we avoid it if we can?

Reply:
We cannot. The report class any almost everything is in `_pytest`, the "public" `pytest` module is nearly empty.

Reply:
Actually, I probably should have a second look round. Maybe pytest exposes the report class somewhere else, but it is definitely defined there.

Reply:
The `pytest.__init__.py` does not expose the `TestReport` class. So sadly the import of `_pytest` is necessary.

Reply:
I `rg`'d the source code of pytest for `TestReport` and it only appeared in `src/_pytest` and the docs and test directories, but not in `src/pytest`. So it is a must.

Reply:
OK, no probs